packages/talos_agent/src/agent.rs

```diff
@@ -83,41 +82,46 @@ impl TalosAgentImpl {
 impl TalosAgent for TalosAgentImpl {
```

Comment:
Please ignore this old agent impl. I am removing it in the next PR which will target this branch.